### PR TITLE
Make our life on GitHub easier:

### PR DESCRIPTION
**Pull request by carstene1ns**
- Automatically [label pull requests](https://github.com/actions/labeler)
- Hilight some warnings in the output
- Keep actions updated

### CODE REVIEW

**Comment by carstene1ns** [comment]
Tested this by merging to my master branch and issuing a pull request:
https://github.com/carstene1ns/easyrpg-player/pull/1
Labels have been applied.

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+
+Building:
+- changed-files:
+  - any-glob-to-any-file: [ '.github/**',
+                            'CMakeLists.txt',
+                            'builds/**',
+                            'Makefile.am',
```
Comment (Ghabry) [inline .github/labeler.yml:7]: Not sure if CMakeLists and Makefile.am should be listed here as any PR that adds new files will trigger this label.

**Comment by Ghabry** [comment]
I have no idea how this Yaml stuff works and do not want to learn it. So feel free to merge it when you consider it ready xD

File: .github/labeler.yml (hunk 1 above)
Comment (carstene1ns) [reply .github/labeler.yml:7]: Technically each added file changes the building process, but I get what you mean.
Unfortunately these labels can only be changed on file basis or branch basis, so only option would be to apply building label manually then.
I think we should just live with some false positives for now and amend the config later, if needed.

**Comment by carstene1ns** [comment]
emote

**Comment by carstene1ns** [comment]
> My guess for the autolabel as Building is `!builds/android/app/**`. This likely matches when any file outside of android is changed 🤔

Oh right, this makes sense. At least is the only negation, so this can happen with the `Building` label.
Will fix in a bit!

**Pull request merged**